src/client/pages/[locale]/installer/03.tsx

```diff
@@ -0,0 +1,154 @@
+import { Captain } from "@/atoms/logo/captain";
+import { makeStaticProperties } from "@/ions/i18n/get-static";
+
+function DummyPrompt() {
+	const { t } = useTranslation(["labels", "texts"]);
+	return (
+		<Box
+			sx={{
+				position: "relative",
+				fontSize: 24,
+				my: "1em",
+				mx: 1,
+				overflow: "hidden",
+				boxShadow: "md",
+			}}
+		>
+			<Input
+				readOnly
+				placeholder={t("labels:placeholder.prompt")}
+				endDecorator={
+					<Box
+						sx={{
+							flex: 1,
+							display: "flex",
+							alignItems: "center",
+							justifyContent: "center",
+							height: 44,
+							width: 44,
+						}}
+					>
+						<Captain sx={{ height: "100%" }} />
+					</Box>
+				}
+				sx={{
+					fontSize: "inherit",
+					lineHeight: 1.25,
+					p: 2,
+					flexDirection: "row",
+				}}
+			/>
+		</Box>
+	);
+}
+
+export default function Page(_properties: InferGetStaticPropsType<typeof getStaticProps>) {
+	const { t } = useTranslation(["labels", "texts"]);
+	const [loading, setLoading] = useState(true);
+	const [starting, setStarting] = useState(false);
+
+	useEffect(() => {
+		const unsubscribeInitialized = window.ipc.on(
+			buildKey([ID.INSTALL], { suffix: ":initialized" }),
+			() => {
+				setLoading(false);
+			}
+		);
+
+		const unsubscribeError = window.ipc.on(buildKey([ID.INSTALL], { suffix: ":error" }), () => {
+			setLoading(true);
+		});
+
+		window.ipc.send(buildKey([ID.INSTALL], { suffix: ":initialize" }));
+
+		return () => {
+			unsubscribeInitialized();
+			unsubscribeError();
+		};
+	}, []);
+
+	return (
+		<AppFrame titleBar={<TitleBar disableMaximize />}>
+			<Box sx={{ minHeight: "100%", display: "flex", flexDirection: "column" }}>
+				<Box sx={{ height: 200, alignContent: "center", px: 2 }}>
+					<DummyPrompt />
+				</Box>
+
+				<Typography level="h1" sx={{ my: 2, textAlign: "center" }}>
+					{t("labels:finishing")}
+				</Typography>
+
+				<Box
+					sx={{
+						flex: 1,
+						display: "flex",
+						flexDirection: "column",
+						justifyContent: "center",
+						mx: 8,
+					}}
+				>
+					<Box
+						sx={{
+							display: "flex",
+							p: 2,
+							backgroundColor: "background.default",
+							borderRadius: "8px",
+							flexDirection: "column",
+						}}
+					>
+						<Typography level="body-lg" sx={{ my: 2, textAlign: "center" }}>
+							{t("texts:howToUseCaptain")}
+						</Typography>
+
+						<Typography level="body-lg" sx={{ my: 2, textAlign: "center" }}>
+							{loading ? t("texts:preparation") : t("texts:preparationDone")}
+						</Typography>
+					</Box>
+				</Box>
+
+				<Box sx={{ display: "flex", justifyContent: "flex-end", m: 1 }}>
+					<Box sx={{ display: "flex", gap: 1 }}>
+						<Button
+							endDecorator={
+								loading || starting ? <CircularProgress size="sm" /> : null
+							}
+							onClick={() => {
+								setStarting(true);
+								window.ipc.send(buildKey([ID.APP], { suffix: ":ready" }), true);
+							}}
+						>
+							{(() => {
```

Comment:
no self calling functions. please use a `let` assignment before the return

Reply:
This was updated!